static/styles/zulip.css

```diff
@@ -143,7 +143,7 @@ p.n-margin {
 
 .fade-in-message {
     animation-name: fadeInMessage;
-    animation-duration: 1s;
+    animation-duration: 1.5s;
```

Comment:
I think you can go a bit more slow here; I feel something between 1.5 to 2s should be great. Can you experiment a bit here which value should look nice?
(At 1.5s I felt it to be too quick and at 2s it was easily noticeable so something in between should give a nice effect)